frontend/talentsearch/src/js/components/request/CreateRequest.tsx

```diff
@@ -157,8 +157,17 @@ export const RequestForm: React.FunctionComponent<RequestFormProps> = ({
       });
   };
 
-  const departmentOptions: Option<string>[] = departments.map(
-    ({ id, name }) => ({
+  const departmentOptions: Option<string>[] = departments
+    .sort((a, b) => {
+      const aName: Maybe<string> = a.name[locale];
+      const bName: Maybe<string> = b.name[locale];
+      if (aName && bName) {
+        return aName.localeCompare(bName);
```

Comment:
Does this take into account strings that begin with an accented character (ex. Élections Canada)? `E` and `É` should probably be sorted as equivalents in French. The [sensitivity](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/localeCompare#using_options) option might be required.

Reply:
I used the `locales` parameter for the `localeCompare`  method in [add current locale to localeCompare](https://github.com/GCTC-NTGC/gc-digital-talent/pull/3110/commits/cfe1424b9d5025f1b77217c41e1f279fb0913d4e) which _should_ and seems to be working.